README.md

```diff
@@ -103,28 +103,32 @@ _Not all options working now!_
     ## if INPUT is ultrastar.txt ##
     default  Creates all
 
+    [separation]
+    # Default is htdemucs
+    --demucs              Model name htdemucs|htdemucs_ft|htdemucs_6s|hdemucs_mmi|mdx|mdx_extra|mdx_q|mdx_extra_q >> ((default) is htdemucs)
+
     [transcription]
     # Default is whisper
-    --whisper               Multilingual model > tiny|base|small|medium|large-v1|large-v2  >> ((default) is large-v2)
+    --whisper               Multilingual model > tiny|base|small|medium|large-v1|large-v2|large-v3  >> ((default) is large-v2)
                             English-only model > tiny.en|base.en|small.en|medium.en
     --whisper_align_model   Use other languages model for Whisper provided from huggingface.co
     --language              Override the language detected by whisper, does not affect transcription but steps after transcription
     --whisper_batch_size    Reduce if low on GPU mem >> ((default) is 16)
     --whisper_compute_type  Change to "int8" if low on GPU mem (may reduce accuracy) >> ((default) is "float16" for cuda devices, "int8" for cpu)
+    --keep_numbers          Numbers will be transcribed as numerics instead of as words >> ((default) is False)
     
     [pitcher]
     # Default is crepe
     --crepe            tiny|full >> ((default) is full)
     --crepe_step_size  unit is miliseconds >> ((default) is 10)
     
     [extra]
-    --hyphenation           Use automatic hyphenation > True|False >> ((default) is True)
-    --disable_separation    Disable track separation > True|False >> ((default) is False)
-    --disable_karaoke       True|False >> ((default) is False)
-    --ignore_audio          True|False >> ((default) is False)
-    --create_audio_chunks   True|False >> ((default) is False)
-    --keep_cache            Keep cache folder after generation > True|False >> ((default) is False)
-    --plot                  Create a pitch plot > True|False >> ((default) is False)
+    --disable_hyphenation   Disable word hyphenation. Hyphenation is enabled by default.
```

Comment:
We need also an update in the Readme in the cli sections in Hyphenate and Seperation

Reply:
Will add that